src/main/groovy/com/deploygate/gradle/plugins/tasks/inputs/DeploymentConfiguration.java

```diff
@@ -40,18 +41,24 @@ public abstract class DeploymentConfiguration {
     @Inject
     public DeploymentConfiguration(
             @NotNull ProviderFactory providerFactory, @NotNull ProjectLayout projectLayout) {
+        // for configuration cache
+        //noinspection Convert2Lambda
         getSourceFilePath()
```

Comment:
https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:requirements

The current limitation... 🤷 The current codebase is not fully compatible with configuration cache build and I'm still wondering if it's possible or not to support configuration cache...